wangle/acceptor/TransportInfo.h

```diff
@@ -24,6 +24,23 @@
 #include <folly/SocketAddress.h>
 #include <folly/portability/Sockets.h>
 
+#ifdef __APPLE__
+#define tcp_info tcp_connection_info
```

Comment:
Not a great idea for a public header to do this with `#define`.

Reply:
I agree. If the `struct tcp_info`s in a couple of other places in the code are changed to `tcp_info`, this can be done with a local `typedef` instead. And the second define can be handled by a local constant.